File: nestjs/ticket-service/src/hello/hello.controller.ts
import { Controller, Get, Logger } from '@nestjs/common';
import { HelloService } from './hello.service';

@Controller()
export class HelloController {
  private readonly logger = new Logger(HelloController.name); // Use default logger

  constructor(private readonly helloService: HelloService) {}

  @Get()
  getHello(): string {
    this.logger.log('Load hello world');
    return this.helloService.getHello();
  }
}

File: spring-boot/tp/tp_archives/store-a/src/main/java/com/imt/intes/controller/HelloController.java
package com.imt.intes.controller;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HelloController {

    private Logger logger = LoggerFactory.getLogger(this.getClass());

    @GetMapping("/hello")
    public String getHelloString () {
        logger.info("Load hello string");
        return "Hello world !";
    }

}
